src/test/regress/spec/isolation_update_node_lock_writes.spec

```diff
@@ -7,8 +7,11 @@ setup
     SELECT create_distributed_table('update_node', 'id');
 }
 
+// we sleep 2 seconds to let isolation test sync metadata
```

Comment:
maybe note that 2 seconds is smaller than citus.metadata_sync_interval (1) for this test